README.md

```diff
@@ -259,15 +258,15 @@ Follow us on Twitter: https://twitter.com/sqlitebrowser
 ## History
 
 This program was developed originally by Mauricio Piacentini
-([@piacentini](https://github.com/piacentini)) from Tabuleiro Producoes, as
+([@piacentini](https://github.com/piacentini)) from Tabuleiro Producoes as
 the Arca Database Browser. The original version was used as a free companion
 tool to the Arca Database Xtra, a commercial product that embeds SQLite
 databases with some additional extensions to handle compressed and binary data.
 
 The original code was trimmed and adjusted to be compatible with standard
 SQLite 2.x databases. The resulting program was renamed SQLite Database
 Browser, and released into the Public Domain by Mauricio. Icons were
-contributed by [Raquel Ravanini](http://www.raquelravanini.com), also from
+contributed by [Raquel Ravanini](http://www.raquelravanini.com) and
```

Comment:
While most of the above are correct, this one really needs to be `from` instead of `and`.  Otherwise it changes the meaning. :smile:

Reply:
Sounds good.  I updated the patch per your feedback.

I'm curious... what is the document trying to convey in that sentence?  That Raquel contributed some icons and there were existing ones from Tabuleiro?  Or something else?  I couldn't quite determine what needed to be conveyed.

Reply:
It's that Raquel contributed icons, while working for Tabuleiro.

At least, that's my recollection of the conversation about it with Mauricio (years ago). :smile:

Reply:
Ah, got it! Thanks!